### DISCUSSION TITLE
.net 6?

### DISCUSSION THREAD

**Discussion by bongomachine** [CommunityToolkit/Maui#870] [Q&A] [answered]
Howdy! What is the future of .Net 6 support? Will bug-fixes be back-ported to version 2.0?

**Comment by VladislavAntonyuk** [comment, marked as answer]
Hi, most likely no. All new features and fixes will be in future versions and target .NET 7 and later.
I understand that net6 is LTS, but I highly recommend you use the latest version of .NET MAUI.
With our version 2.0, we updated to the latest .NET MAUI (6.0.541 at the moment of release).
Sorry, if my comment disappoints you.

**Reply by bongomachine** [reply]
I see. Unfortunately I have external dependencies on .net 6 and therefore I'm not able to upgrade.

**Reply by pictos** [reply]
If your project targets .NET 7 you should be able to consume .NET 6 packages

**Reply by bongomachine** [reply]
Ok, I'll look into it. Thanks!

**Comment by CliffAgius** [comment]
Closed as answered
